recipes/py-mathjax/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "py-mathjax" %}
+{% set version = "2.7.5" %}
+{% set pypi_build = ".1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
```

Comment:
Ditto here on the version string. If this is not correct it can be fixed in the feedstock.